src/msw/printdlg.cpp

```diff
@@ -76,6 +76,24 @@ class WinPrinter
         return result;
     }
 
+    wxMemoryBuffer GetData(int level) {
```

Comment:
There are several minor formatting issues in this change, e.g. here
```suggestion
    wxMemoryBuffer GetData(int level)
    {
```

Reply:
Fixed with my latest commit.